tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.py

```diff
@@ -48,16 +116,41 @@ def get_configuration(request):
                          ])
 def test_max_files_per_second(inode_collision, get_configuration, configure_environment, restart_syscheckd,
                               wait_for_fim_start):
-    """ Check that FIM sleeps for one second when the option max_files_per_second is enabled
-
-    Args:
-        inode_collision (boolean): Signals if the test should check the limit while running inode collisions.
-        get_configuration (fixture): Gets the current configuration of the test.
-        configure_environment (fixture): Configure the environment for the execution of the test.
-        restart_syscheckd (fixture): Restarts syscheck.
-        wait_for_fim_start (fixture): Waits until the first FIM scan is completed.
-    Raises:
-        TimeoutError: If an expected event couldn't be captured.
+    """ 
+    description: 
+        Check that FIM sleeps for one second when the option max_files_per_second is enabled
+    
+    wazuh_min_verion:
+        4.2
```

Comment:
This needs to be added in the future with more data because we don't know what the minimum supported version is.